src/redux/features/taskBoardSlice.tsx

```diff
@@ -71,6 +71,12 @@ const taskBoardSlice = createSlice({
     },
     setViewSettings: (state, action: { payload: CreateViewSettingsDTO }) => {
       state.view = action.payload.viewMode
+      if (action.payload.filterOptions.assignee) {
+        const assigneeCheck = state.assignee.find((assignee) => assignee.id == action.payload.filterOptions.assignee)
+        if (!assigneeCheck) {
+          action.payload.filterOptions.assignee = ''
+        }
+      }
```

Comment:
I am unaware of the context here but looking at the code, feels like this should be handled in separate action, if the logic is within the context LGTM.

Reply:
I think if we handled this outside the redux slice, there will be extra checks and fetches. So I think it is better to handle this in the taskBoardSlice itself because other states are easily accessible since filterOptions state depends on assigneeList. 

cc : @aatbip @rrojan

Reply:
@arpandhakal This seems like a case we could frequently come across in the future. It makes sense to refactor this to a separate action imo too.

Reply:
@arpandhakal i make sense to keep in the slice itself, I'm suggesting only a different action within the slice

Reply:
sure. thanks. I misunderstood 😅